config/api.yml

```diff
@@ -35,6 +35,13 @@
     - :post
     - :patch
     - :delete
+    :gpppd: &gpppdo
```

Comment:
This is keyed the same as the previous one - I'm surprised this doesn't cause an error!